src/Console/Application.php

```diff
@@ -183,17 +182,10 @@ protected function initializeComposerHelper(): Helper\ComposerHelper
             return $this->composerHelper;
         }
 
-        try {
-            $composerFile = getcwd().DIRECTORY_SEPARATOR.'composer.json';
-            $configuration = Composer::loadConfiguration();
-            Composer::ensureProjectBinDirInSystemPath($configuration->get('bin-dir'));
-            $rootPackage = Composer::loadRootPackageFromJson($composerFile, $configuration);
-        } catch (RuntimeException $e) {
-            $configuration = null;
-            $rootPackage = null;
-        }
+        $composerFileLocation = getcwd().DIRECTORY_SEPARATOR.'composer.json';
```

Comment:
Is this path always guaranteed correct?
if it can't find the composer.json file, grumphp will crash on `file_get_contents`

Reply:
No it is not always available. probably better to detect composer.json from current directory and from all parents until one is found?

Reply:
There is a check in ComposerFile.php on line 21

Reply:
In the old codebase, there was a catch which returns an empty composer info.
Maybe it's best to catch the error from ComposerFile line 21 and also return an empty Composer instance?
That way, the code keeps on working if there is no composer.json in the cwd (which probably will happen more then once... ;) )